ansible/roles/dev/tasks/main.yml

```diff
@@ -175,3 +175,23 @@
 - name: Install pulp-dev library (system-wide)
   command: pip3 install -e {{pulp_devel_dir}}/devel/pulp-dev
   when: "{{ ['devel'] | issubset(pulp_available_repositories) }}"
+
+- name: install systemd files
+  copy:
```

Comment:
I haven't noticed your comment, but I came to the same conclusion. And tested it now it works.